examples/user_interfaces/svg_tooltip.py

```diff
@@ -31,41 +31,22 @@
 fig, ax = plt.subplots()
 
 # Create patches to which tooltips will be assigned.
-circle = plt.Circle((0, 0), 5, fc='blue')
-rect = plt.Rectangle((-5, 10), 10, 5, fc='green')
-
-ax.add_patch(circle)
-ax.add_patch(rect)
-
-# Create the tooltips
-circle_tip = ax.annotate(
-    'This is a blue circle.',
-    xy=(0, 0),
-    xytext=(30, -30),
-    textcoords='offset points',
-    color='w',
-    ha='left',
-    bbox=dict(boxstyle='round,pad=.5', fc=(.1, .1, .1, .92),
-              ec=(1., 1., 1.), lw=1, zorder=1))
-
-rect_tip = ax.annotate(
-    'This is a green rectangle.',
-    xy=(-5, 10),
-    xytext=(30, 40),
-    textcoords='offset points',
-    color='w',
-    ha='left',
-    bbox=dict(boxstyle='round,pad=.5', fc=(.1, .1, .1, .92),
-              ec=(1., 1., 1.), lw=1, zorder=1))
-
-# Set id for the patches
-for i, t in enumerate(ax.patches):
-    t.set_gid('patch_%d' % i)
-
-# Set id for the annotations
-for i, t in enumerate(ax.texts):
-    t.set_gid('tooltip_%d' % i)
+rect1 = plt.Rectangle((10, -20), 10, 5, fc='blue')
```

Comment:
I wasn't able to find a clean way to iterate the coordinates of a circle in the example for loop.  That's the only reason.